### PR TITLE
update px version to 2.6

### PR DESCRIPTION
**Pull request by trierra**

### CODE REVIEW

File: charts/portworx/v1.1.1/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  - portworx
+  - amd64
+home: https://portworx.com/
+maintainers:
+  - name: harsh-px
+    email: harsh@portworx.com
+  - name: trierra
+    email: oksana@portworx.com
+  - name: sharma-tapas
+    email: tapas@portworx.com
```
Comment (harsh-px) [inline charts/portworx/v1.1.1/Chart.yaml:21]: update emails

File: charts/portworx/v1.1.1/values.yaml [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+csi: false                            # Enable CSI
+
+serviceAccount:
+  hook:
+    create: true
+    name:
+
+deploymentType: oci                   # accepts "oci" or "docker"
+imageType: none                       #
+imageVersion: 2.6                     # Version of the PX Image.
```
Comment (harsh-px) [inline charts/portworx/v1.1.1/values.yaml:44]: wrong image version. Needs to be 2.6.1.4

File: charts/portworx/v1.1.1/values.yaml [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+platformOptions: none                 # AKS, EKS or GKE platforms
+
+customRegistryURL:
+registrySecret:
+
+clusterName: mycluster                # This is the default. please change it to your cluster name.
+secretType: k8s                      # Defaults to None, but can be AWS / KVDB / Vault.
+envVars: none                         # NOTE: This is a ";" seperated list of environment variables. For eg: MYENV1=myvalue1;MYENV2=myvalue2
+stork: true                           # Use Stork https://docs.portworx.com/scheduler/kubernetes/stork.html for hyperconvergence.
+storkVersion: 2.4.3
```
Comment (harsh-px) [inline charts/portworx/v1.1.1/values.yaml:27]: install.portworx.com/2.6 is using Stork 2.5.0

File: charts/portworx/v1.1.1/questions.yml [hunk 4]
```diff
@@ -0,0 +1,921 @@
...
+  group: "Advanced parameters"
+
+- variable: envVars
+  label: "Environment variables"
+  description: "Enter your environment variables separated by semicolons (example: API_SERVER=http://lighthouse-new.portworx.com;MYENV1=val1;MYENV2=val2). These environment variables will be exported to Portworx."
+  type: string
+  group: "Advanced parameters"
+
+- variable: imageVersion
+  default: "2.6"
```
Comment (harsh-px) [inline charts/portworx/v1.1.1/questions.yml:912]: fix stork and px versions in this file too

**Review by harsh-px — CHANGES_REQUESTED** [review]

**Comment by PennyScissors** [comment]
Hello @trierra @harsh-px, I'm working on cleaning up this repository. Is this issue still relevant? I see that there hasn't been any activity in a while.

**Comment by harsh-px** [comment]
This can be closed

**Comment by PennyScissors** [comment]
Closing in favor of https://github.com/rancher/charts/pull/1236

**Pull request closed without merging**